src/App.tsx

```diff
@@ -31,11 +31,18 @@ function AppRoot() {
     setShowCreateVFEForm(false);
   }
 
+  function handleUpdateVFE(updatedVFE: VFE) {
+    console.log("Updated VFE (in AppRoot):", updatedVFE);
```

Comment:
Not sure how everyone feels but we might want to remove the console.logs before we merge to main